Content.Server/Explosion/Components/OnUseTimerTriggerComponent.cs

```diff
@@ -32,5 +32,12 @@ public sealed class OnUseTimerTriggerComponent : Component
 
         [DataField("beepParams")]
         public AudioParams BeepParams = AudioParams.Default.WithVolume(-2f);
+
+        /// <summary>
+        ///     Should timer be started when it was stuck to another entity.
+        ///     Used for C4 charges and similar behaviour.
+        /// </summary>
+        [DataField("startOnStick")]
+        public bool StartOnStick;
```

Comment:
this feels a little strange to have on this component since its so specific but honestly its fine

Reply:
Well, timer need to somehow get event that it was stuck into entity to start timing. I could create new component like `StartTimerWhenStuck`, but it doesn't worth it imo.